# Generated by Django 5.1.7 on 2025-03-27 20:36

from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='LogEntry',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('timestamp', models.DateTimeField()),
                ('level', models.CharField(choices=[('DEBUG', 'Debug'), ('INFO', 'Info'), ('WARNING', 'Warning'), ('ERROR', 'Error'), ('CRITICAL', 'Critical')], max_length=10)),
                ('logger', models.CharField(max_length=100)),
                ('module', models.CharField(max_length=100)),
                ('function', models.CharField(max_length=100)),
                ('message', models.TextField()),
                ('request_method', models.CharField(blank=True, max_length=10, null=True)),
                ('request_path', models.CharField(blank=True, max_length=255, null=True)),
                ('request_status_code', models.IntegerField(blank=True, null=True)),
                ('request_client_ip', models.GenericIPAddressField(blank=True, null=True)),
                ('request_user', models.CharField(blank=True, max_length=100, null=True)),
                ('exception_type', models.CharField(blank=True, max_length=100, null=True)),
                ('exception_message', models.TextField(blank=True, null=True)),
                ('exception_stack_trace', models.TextField(blank=True, null=True)),
                ('extra_data', models.JSONField(blank=True, null=True)),
            ],
        ),
    ]
